### PR TITLE
Limit browser window size

### PR DESCRIPTION
**Pull request by pranavkarthik10**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [x] Feature implementation
- [ ] Doc updates

* **What changes have you introduced?**
Added a new attribute within the BrowserWindow initialization to limit the width and height to a minimum.

* **Does this PR introduce a breaking change?**
No

* **Preview / Steps to verify your work**:
As can be viewed, the minimizing capability is limited after a certain limit.
Screen Recording 2020-01-03 at 11 52 29 AM

### CODE REVIEW

**Comment by pranavkarthik10** [comment]
@mariobehling Please review and merge this! Thank you.

**Comment by Sid911** [comment]
@pranavkarthik10 I think you should reduce the size of min height and width... most of the time pslab is used with small portable monitors ... plus responsiveness of the webapp is being improved in recent pr's you can check them.. so screen size could theoretically be pretty small and this should work. Although minimum is required....

**Comment by pranavkarthik10** [comment]
Thanks @Sid911 . Updated it with a smaller min size. I'm sure this can fit within smaller monitors/displays as well.

**Comment by pranavkarthik10** [comment]
@adityastic @cweitat Can you please review this and merge

**Comment by pranavkarthik10** [comment]
@fcartegnie @cweitat @adityastic Please take a look and merge! Thank you.

File: src/screen/AboutUs/AboutUs.styles.js [hunk 1]
```diff
@@ -11,8 +11,8 @@ export const Container = styled.div`
 
 export const Wrapper = styled(Card)`
   margin: 16px 0px;
+  width: 35%;
   height: 870px;
-  width: 95%;
```
Comment (fcartegnie) [inline src/screen/AboutUs/AboutUs.styles.js]: reverting code from previous merge ???????

File: public/electron.js [hunk 2]
```diff
@@ -30,17 +30,13 @@ function createWindow() {
         protocol: 'file:',
         slashes: true,
       });
-<<<<<<< HEAD
```
Comment (fcartegnie) [inline public/electron.js]: This is not acceptable

File: src/screen/AboutUs/AboutUs.styles.js (hunk 1 above)
Comment (pranavkarthik10) [reply src/screen/AboutUs/AboutUs.styles.js]: This is to fix merge problems so it is up to date with the current version of this repository.

File: public/electron.js (hunk 2 above)
Comment (pranavkarthik10) [reply public/electron.js]: Sorry about that, it is removed in the latest commit.

**Comment by pranavkarthik10** [comment]
@fcartegnie Sure, I will create a new PR then. Thanks for your help.

**Comment by pranavkarthik10** [comment]
I have created the new PR here: https://github.com/fossasia/pslab-desktop/pull/575

**Pull request closed without merging**